app/controllers/vehicles_controller.rb

```diff
@@ -34,7 +34,7 @@ def edit
   def update
     if @vehicle.update(vehicle_params)
       @vehicle.save
```

Comment:
No need to call `.save` since `.update` already does it
See here : [Update](https://guides.rubyonrails.org/active_record_basics.html#update)